### PR TITLE
changed some variable declarations in Leaflet.DistortableImage/src/components/DistortableImage.Keymapper.js 

### PR DESCRIPTION
**Pull request by jesutobi**
@TildaDares I have fixed the issues in  ..... Fixes #1086 
[https://github.com/publiclab/Leaflet.DistortableImage/issues/1086](url).

in this directory Leaflet.DistortableImage/src/components/DistortableImage.Keymapper.js 
I changed the variable declaration on line 42 from "var" to const 
I changed the variable declaration on line 45 from "var" to let 
I changed the variable declaration on line 52 from "var" to const 
I changed the variable declaration on line 66 from "var" to const 
I changed the variable declaration on line 131 from "var" to const

### CODE REVIEW

**Review by PeculiarE — CHANGES_REQUESTED** [review]
Hi @jesutobi....thank you for working on this. Could you please undo the changes in the `leaflet.distortableimage.js` file. Only the `DistortableImage.Keymapper.js` file should be changed. Thank you!

**Comment by jesutobi** [comment]
@PeculiarE i have done it as you requested

File: src/components/DistortableImage.Keymapper.js [hunk 1]
```diff
@@ -39,17 +39,17 @@ L.DistortableImage.Keymapper = L.Handler.extend({
   },
 
   _buildContainer() {
-    var container = L.DomUtil.create('div', 'ldi-keymapper-hide');
+    const container = L.DomUtil.create('div', 'ldi-keymapper-hide');
     container.setAttribute('id', 'ldi-keymapper');
 
-    var divider = L.DomUtil.create('br', 'divider');
+    let divider = L.DomUtil.create('br', 'divider');
```
Comment (PeculiarE) [inline src/components/DistortableImage.Keymapper.js:45]:
```suggestion
    const divider = L.DomUtil.create('br', 'divider');
```

**Review by PeculiarE — CHANGES_REQUESTED** [review]

**Comment by PeculiarE** [comment]
Hi @jesutobi....please accept and commit the suggested changes by clicking on the `Commit suggestion` button in the review above.

Once this is done, your PR will be ready to merge.


Thank you, and great work so far!

**Comment by jesutobi** [comment]
@PeculiarE thank you, i have  added the suggested change

File: src/components/DistortableImage.Keymapper.js [hunk 2]
```diff
@@ -128,7 +128,7 @@ L.DistortableImage.Keymapper = L.Handler.extend({
   _injectIconSet() {
     if (document.querySelector('#keymapper-iconset')) { return; }
 
-    var el = L.DomUtil.create('div', '');
+   const el = L.DomUtil.create('div', '');
```
Comment (PeculiarE) [inline src/components/DistortableImage.Keymapper.js:131]:
```suggestion
    const el = L.DomUtil.create('div', '');
```

**Review by PeculiarE — CHANGES_REQUESTED** [review]

**Comment by PeculiarE** [comment]
Thank you @jesutobi...but the build is failing due to indentation issues. An extra space was omitted in the initial commit.

So, once again, please accept and commit the suggested change by clicking on the Commit suggestion button in the review above, and then we can merge 😄

**Comment by jesutobi** [comment]
@PeculiarE thank you, i am sorry for the many bugs 😥, i just commited the suggested  changes

**Comment by PeculiarE** [comment]
It's okay @jesutobi....thanks for sticking with it ❤️

**Comment by jesutobi** [comment]
@PeculiarE, i still see the "merging is blocked" is there something else i didnt take note off ?

**Review by PeculiarE — APPROVED** [review]
Well done once again! @jesutobi

@TildaDares ....this is ready to merge. Thank you!

**Comment by PeculiarE** [comment]
> @PeculiarE, i still see the "merging is blocked" is there something else i didnt take note off ?

Oh, don't worry about it. Merging can only be done by those with the appropriate repo permissions

**Comment by jesutobi** [comment]
> It's okay @jesutobi....thanks for sticking with it ❤️

thanks you

**Comment by jesutobi** [comment]
> It's okay @jesutobi....thanks for sticking with it ❤️

thanks you

**Comment by jesutobi** [comment]
> > @PeculiarE, i still see the "merging is blocked" is there something else i didnt take note off ?
> 
> Oh, don't worry about it. Merging can only be done by those with the appropriate repo permissions

okay

**Comment by TildaDares** [comment]
Congratulations on merging your PR @jesutobi 🎉

**Comment by TildaDares** [comment]
Thank you for the reviews @PeculiarE!

**Comment by 7malikk** [comment]
Congratulations @jesutobi 🎊

**Comment by jesutobi** [comment]
> Congratulations on merging your PR @jesutobi 🎉

thank you :)

**Comment by jesutobi** [comment]
> Congratulations @jesutobi 🎊

thanks

**Pull request merged**